### PR TITLE
Cmake find opus file cmake

### PR DESCRIPTION
**Pull request by madebr**
Addresses https://github.com/libsdl-org/SDL_mixer/issues/467

### CODE REVIEW

File: SDL2_mixerConfig.cmake.in [hunk 1]
```diff
@@ -82,7 +82,11 @@ include(CMakeFindDependencyMacro)
     endif()
 
     if(SDL2MIXER_VORBIS_VORBISFILE AND NOT SDL2MIXER_VENDORED AND NOT TARGET vorbisfile::vorbisfile)
```
Comment (FtZPetruska) [inline SDL2_mixerConfig.cmake.in:84]: Since the package is `Vorbis`, shouldn't the target checked be `Vorbis::vorbisfile` ?

Comment (madebr) [reply SDL2_mixerConfig.cmake.in:84]: You're right. This was the last `Vorbis::vorbisfile`.

Comment (madebr) [reply SDL2_mixerConfig.cmake.in]: Fixed in 6825230

**Comment by sezero** [comment]
What else is left for this to be merged, except for a rebase and fixing conflicts?

**Comment by madebr** [comment]
> What else is left for this to be merged, except for a rebase and fixing conflicts?

Nothing. Squash and merge please :) (after ci passes)

**Pull request merged**